Prototope/Gesture.swift

```diff
@@ -268,6 +268,79 @@ public class PanGesture: GestureType {
 	}
 }
 
+/** A rotation gesture recognizes a standard iOS rotation: it doesn't begin until the user's moved some number of points, then it tracks new touches coming and going over time as well as rotation relative to the beginning of the gesture and the current rotation velocity. It exposes simple access to the path of the center of all the touches and the rotation and rotation velocity values. */
+public class RotationGesture: GestureType {
+    /** The handler will be invoked as the gesture recognizes and updates; it's passed the gesture's current
+    phase (see ContinuousGesturePhase documentation), rotation relative to the state at the beginning 
+    of the gesture in radians, rotation velocity in radians per second and also a touch sequence representing the center of all the touches involved in the rotation gesture. */
+    public convenience init(_ handler: (phase: ContinuousGesturePhase, rotationRadians: Double, rotationVelocity: Double, centroidSequence: TouchSequence<Int>) -> ()) {
```

Comment:
Wish there were some good way to make defining this closure handler more friendly. This many positional arguments is rough!

One idea: what if we defined a `RotationGesture.State` `struct` subtype (with `phase`, `rotationRadians`, etc fields) and passed one of those? Then you could do lightweight things like this:

```
layer.gestures.append(RotationGesture { layer.rotationRadians = $0.rotationRadians })
```